public/css/main.css

```diff
@@ -100,6 +114,22 @@ li {
   --color-workspace-item-focus-mg: var(--color-gray-lighten-50);
   --color-workspace-item-focus-bg: var(--color-gray-lighten-55);
 
+  --color-modal-fg: var(--color-main-fg);
+  --color-modal-mg: var(--color-gray-lighten-60);
+  --color-modal-bg: var(--color-gray-lighten-100);
+  --color-modal-inner-border: var(--color-gray-lighten-50);
+  --color-modal-separator: var(--color-gray-lighten-55);
+  --color-modal-shadow: rgba(24, 24, 28, 0.2); /* 20% gray-darken-30 */
+  --color-modal-overlay: rgba(24, 24, 28, 0.5); /* 50% gray-darken-30 */
```

Comment:
At some point we'll need to do a color variable overhaul such that semi transparent colors like this can be based on a variable.